modules/programs/nushell.nix

```
@@ -121,6 +121,16 @@ in {
         this option) to command strings or directly to build outputs.
       '';
     };
+
+    envVars = mkOption {
+      type = types.attrsOf types.str;
+      default = { };
+      example = { FOO.text = "$'(echo BAR)'"; };
```

```suggestion
      example = { FOO = "$'(echo BAR)'"; };
```
I'd also expect a more straightforward example without command substitution.

---

I wanted to convey the fact it was shell expanded. I'm happy to go with something simpler though.